<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>532</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QListView" name="lwAllBibtex">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Pesquisa</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLineEdit" name="leQuery"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbSearch">
         <item>
          <property name="text">
           <string>Mostrar Todos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Autor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ano</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Veículo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Título</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSearch">
         <property name="text">
          <string>Filtrar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="lbReference">
           <property name="text">
            <string>Referência</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leReference"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbAuthor">
           <property name="text">
            <string>Autor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leAuthor"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbTitle">
           <property name="text">
            <string>Título</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leTitle"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbYear">
           <property name="text">
            <string>Ano</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leYear"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lbPublisher">
           <property name="text">
            <string>Editora</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePublisher"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lbJournal">
           <property name="text">
            <string>Periódico</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leJournal"/>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lbVolume">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leVolume"/>
         </item>
        </layout>
       </item>
       <item row="7" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lbNumber">
           <property name="text">
            <string>Número</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leNumber"/>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lbPages">
           <property name="text">
            <string>Páginas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePages"/>
         </item>
        </layout>
       </item>
       <item row="9" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lbBookTitle">
           <property name="text">
            <string>Título do Livro</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leBookTitle"/>
         </item>
        </layout>
       </item>
       <item row="10" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>79</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="pbSave">
         <property name="text">
          <string>Salvar</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="pbRemove">
         <property name="text">
          <string>Remover</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>layoutWidget</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>layoutWidget_2</zorder>
      <zorder>layoutWidget_3</zorder>
      <zorder>layoutWidget_4</zorder>
      <zorder>layoutWidget_5</zorder>
      <zorder>layoutWidget_6</zorder>
      <zorder>layoutWidget_7</zorder>
      <zorder>layoutWidget_8</zorder>
      <zorder>layoutWidget_9</zorder>
      <zorder>verticalSpacer</zorder>
      <zorder>pbSave</zorder>
      <zorder>pbRemove</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="action_Sair"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Sair">
   <property name="text">
    <string>&amp;Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
